doc/security/cryptolib/cryptolib_api.md

```diff
@@ -682,7 +686,7 @@ break a cryptographic algorithm. Security strength of an algorithm with
 key length 'k'is expressed in "bits" where n-bit security means that the
 attacker would have to perform 2^n^ operations to break the algorithm.
 
-The table below summarizes the security strength for all the .
+The table below summarizes the security strength for the supported [cryptographic algorithms](#openTitan-supported-crypto-algorithms-and-modes).
```

Comment:
Thanks for catching this, the docs-to-markdown conversion deleted all the links and I must have missed this one!